mix/mix_protocol.nim

```diff
@@ -352,6 +399,7 @@ proc `$`*(d: MixDestination): string =
 
 proc anonymizeLocalProtocolSend*(
     mixProto: MixProtocol,
+    incoming: AsyncQueue[seq[byte]],
```

Comment:
Similar to comment in `buildSurbs`:
- if `incoming` is always a fresh queue, might be simpler to create  internally here too?